jest.config.js

```diff
@@ -16,6 +16,6 @@ module.exports = {
     testRegex: '/__tests__/.*test\\.ts$',
     testTimeout: 1000,
     transform: {
-        '^.+\\.ts$': ['ts-jest', { compiler: 'ttypescript' }],
+        '^.+\\.ts$': ['ts-jest'],
```

Comment:
We only ever used `ttypescript` to allow `ts-jest` to pick up `@nestjs/swagger` logic that was triggered by compile-time transformers. 

 - We don't rely on transformer logic in our libraries, generally.
 - We no longer need this logic because newer versions of `ts-jest` have a proper transformers hook.